Gemfile

```diff
@@ -42,6 +43,8 @@ group :development do
   gem 'listen', '~> 3.3'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'guard', '~> 2.18.0'
+  gem 'rexml'
```

Comment:
Is this gem needed?

Reply:
Yes, without `rexml` i get some errors.

Reply:
As we saw, `rexml` seems to be needed only if we run the tests through `rake`. `bundle exec guard` seems to work fine without them.

We'll therefore remove this gem & add it later if errors arise.